### PR TITLE
Update issue template to point to Jira

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -11,7 +19,9 @@ WDL https://gatkforums.broadinstitute.org/wdl/categories/ask-the-wdl-team
 CWL https://www.biostars.org/
 -->
 
-<!-- Are you seeing something that looks like a bug?  Then great!  You're in the right place.  -->
+<!-- Are you seeing something that looks like a bug?  Then great!  You're almost in the right place.  -->
+
+<!-- You'll want to go to https://broadworkbench.atlassian.net/projects/BA/issues and then tell us: -->
```
Comment (cjllanwarne) [inline .github/issue_template.md]: We should probably remove all this "remember to include" cruft while we're here. We can always make them JIRA fields instead

Comment (kshakir) [reply .github/issue_template.md]: Have a link?

Briefest of googling, I only found a way to do templates at ~$10 per Jira user.

**Comment by gemmalam** [comment]
I think this is great! I would like @ruchim to take a look and see what she thinks about turning off issue creation after a certain date. Also, if we should close out all of the existing tickets after that date. Jira is open for folks to join and look at our board so this shouldn't be too big of a pain, but I don't know how we would get the message out to everyone.

File: .github/issue_template.md (hunk 1 above)
Comment (cjllanwarne) [reply .github/issue_template.md]: 🤢

File: .github/issue_template.md [hunk 2]
```diff
@@ -1,7 +1,15 @@
+###
+### IMPORTANT: Please file new issues over in our Jira issue tracker!
+###
+### https://broadworkbench.atlassian.net/projects/BA/issues
+###
+### You may need to create an account before you can view/create issues.
```
Comment (salonishah11) [inline .github/issue_template.md]: TOL: Should we add that they can use their Gmail account for the same purpose?

File: .github/issue_template.md (hunk 1 above)
Comment (danbills) [reply .github/issue_template.md]: QA has it set up for prod issues, so if they can do I'm guessing we can too

File: .github/issue_template.md (hunk 2 above)
Comment (cjllanwarne) [reply .github/issue_template.md]: Oops, I missed this before merging. Could be a good follow up though (but is it true? I thought they could create an account *using* their gmail, but not sign in with it? Maybe my confusion is exactly why we should add something!)

Comment (aednichols) [reply .github/issue_template.md]: You can sign in with any Google account, there is no user facing "account creation" sequence per se though obviously behind the scenes JIRA is registering you in its user database

**Pull request merged**